confluent_server/confluent/plugins/hardwaremanagement/ipmi.py

```diff
@@ -762,6 +764,50 @@ def handle_identifier(self):
             self.ipmicmd.set_mci(mci)
             return
 
+    def handle_ntp(self):
+        if self.element[3] == 'enabled':
+            if 'read' == self.op:
+                enabled = self.ipmicmd.get_ntp_enabled()
+                self.output.put(msg.NTPEnabled(self.node, enabled))
+                return
+            elif 'update' == self.op:
+                enabled = self.inputdata.ntp_enabled(self.node)
+                if enabled == 'on':
+                    enabled = True
+                else:
+                    enabled = False
+                self.ipmicmd.set_ntp_enabled(enabled)
+                return
+        elif self.element[3] == 'servers':
+            if len(self.element) == 4:
+                self.output.put(msg.ChildCollection('all'))
+                self.output.put(msg.ChildCollection('1'))
+                self.output.put(msg.ChildCollection('2'))
```

Comment:
Might be nice to actually use the length of self.ipmicmd.get_ntp_servers() to control how many current ones are listed